### ISSUE TITLE
IE 8 Schedule Viewer compatibility

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#229]
```
I've seen a few reports that the schedule viewer doesn't work in IE 8. Could 
someone with a Windows machine take a look and see if it works, and if not 
try to work out why?
```

Original issue reported on code.google.com by `tom.brow...@gmail.com` on 23 Mar 2010 at 10:20
- Merged into: #208

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Could http://code.google.com/p/googletransitdatafeed/issues/detail?id=208 be 
the 
problem you saw mentioned?

This fix was not included in 1.2.5, so if that's the version they are using 
that might 
be problem. The workaround is to select compatibility view - the "broken page" 
button between the address bar and the refresh button. See 
http://support.microsoft.com/library/images/support/kbgraphics/PUBLIC/EN-
US/956157a.png

If this is not the problem do you have any more information that might be of 
help?

Thanks,
- André
```

Original comment by `a...@google.com` on 23 Mar 2010 at 11:45

**Comment by GoogleCodeExporter** [comment]
```
Creating this was silly of me. Thanks André.
```

Original comment by `tom.brow...@gmail.com` on 24 Mar 2010 at 12:21
- Changed state: **Duplicate**
